config/ePO-default/config-parameters.xsl

```diff
@@ -1,22 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
-    xmlns:xs="http://www.w3.org/2001/XMLSchema"
-    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
-    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" 
+<xsl:stylesheet 
```

Comment:
it is better if there are not namespaces defined in any config file

Reply:
For the moment, I prpose that we keep like that. We will improve it later since it works.